extensions/inkeep/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "inkeep",
+  "title": "Inkeep",
+  "description": "Interact with Inkeep Completions API to get AI-powered answers from your knowledge base.",
+  "icon": "icon.png",
+  "author": "marcklingen",
+  "categories": [
+    "Productivity",
+    "Developer Tools",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "ask-inkeep",
+      "title": "Inkeep",
```

Comment:
```suggestion
      "title": "Ask Inkeep",
```
Could it be something like this?

Reply:
Thought about this but "Ask" seems to be prefix for AI tools and I wanted to make this more clear

![CleanShot 2025-03-03 at 11 27 21](https://github.com/user-attachments/assets/4ed2f8d5-c611-4171-a8d6-55a6de3476c9)

Open for feedback on this!

Reply:
What does it list? We could use `Search Inkeep` maybe?

Reply:
"Search" sounds good, will update this

Reply:
Done